### DISCUSSION TITLE
Apple pencil only drawing mode

### DISCUSSION THREAD

**Discussion by torazine** [keiwando/toonsquid#5] [Ideas]
Is there a setting that can only be drawn with an Apple pencil?
If not, please make that feature

**Comment by keiwando** [comment]
Hi!
There currently is no such option but I can add it with one of the next updates.
Thank you for your suggestion!

**Reply by torazine** [reply]
Thank you for answer
Thank you again for creating a great app

**Reply by keiwando** [reply]
This setting is now available in the 1.6 update
